### DISCUSSION TITLE
ToonSquid 2.0

### DISCUSSION THREAD

**Discussion by DinoAwakening** [keiwando/toonsquid#475] [General]
I don’t think ToonSquid would lead a Minecraft-like sequence for updates. So, since 1.10 dropped, I think the next update that might come up now is 2.0. Let’s listen some theories or wishes from the community! :D

I’d really like and improvement for Bucket Fill Tool ;)

**Comment by aluminum1** [comment]
I would love to have "rotate object along path" option for motion paths. This makes perfect sense for smooth Bezier paths, but it makes sense for polygonal paths too, the rotation would just happen instantaneously at the vertices, like pacman when he turns a corner.
